packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -165,6 +176,10 @@ export function trackViews(
       currentView.addTiming(name, time)
       currentView.scheduleUpdate()
     },
+    addFeatureFlags: (featureFlags: Context) => {
+      currentView.addFeatureFlags(featureFlags)
+      currentView.scheduleUpdate()
+    },
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏IMO, trackViews has enough responsibility related to view tracking logic. What about extracting feature flag management to a dedicated component? maybe something like `urlContexts`

Reply:
Yes, I also love standalone module :). With this approach the only complexity I see it to trigger a `view_updated` when a new feature flag is added. 
I could:
- a make a featureFlagObservable so that the view module is notified  
- expose the scheduleUpdate() method to the featureFlagContexts
Wdyt?

Reply:
I guess it was to be consistent to the timing updates behaviour since the implementation is currently very close. But it might not be needed.

Reply:
I'll create a follow up task for this update